### DISCUSSION TITLE
`x-model.fill` doesn't support textareas

### DISCUSSION THREAD

**Discussion by saveman71** [alpinejs/alpine#3547] [7. Bugs (archived)] [answered]
I'm not sure if it's a bug or intended, but [x-model.fill](https://alpinejs.dev/directives/model#fill) doesn't support textareas at the moment. I feel like it was just an oversight of the original implementation, as textareas work quite differently under the hood.

Is this a bug?

There's this PR https://github.com/alpinejs/alpine/pull/3495 that adds support for selects.

**Comment by ekwoka** [comment, marked as answer]
As discussed in #3495 that change will support everything that `x-model` already works with.
